lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -559,6 +570,7 @@ public void OnNext(IFailedEvaluator failedEvaluator)
                             break;
 
                         case SystemState.Fail:
```

Comment:
This is not expected case. Because when we set system state to Fail, we throw exception.